static/sass/_pattern_cards.scss

```diff
@@ -109,19 +111,11 @@
       }
     }
 
-    &.is-selected {
-      background-color: $color-brand;
-      color: $color-x-light;
-
-      &:hover {
-        background-color: $color-mid-dark;
-      }
-    }
-
     &.is-empty {
       background-color: transparent;
-      border: 1px dashed $color-x-light;
+      border: 1px dashed #757575;
```

Comment:
Should we house this color in a variable instead?

Reply:
Good suggestion! Thank you @mtruj013 !